CMakeLists.txt

```diff
@@ -156,6 +157,10 @@ endif()
 SET(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g")
 SET(CMAKE_CXX_FLAGS_DEBUG  "-O0 -g")
 
+if (NGRAPH_CODE_COVERAGE_ENABLE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --coverage")
```

Comment:
Does CMAKE_CXX_FLAGS get used during linking?  I believe that "--coverage" must be used for both compilations commands and the linking command.